data/json/items/gun/50.json

```diff
@@ -4,7 +4,7 @@
     "copy-from": "rifle_semi",
     "looks_like": "modular_ar15",
     "type": "GUN",
-    "name": { "str_sp": "Barrett M107A1" },
+    "name": { "str_sp": "Barrett M107A1 rifle" },
```

Comment:
Considering we differentiated “sniper rifles” from regular “rifles” and “assault rifles” before for sake of visual recognition, it might be prudent to term this an “anti-material rifle.”